test/document_test.rb

```diff
@@ -348,6 +364,15 @@ def setup
     assert_equal 'bar', test_foo.doc['default_test_property']
   end
 
+  test 'created at is current time' do
+    resp = {ok: true, id: "with_time/timed", rev: "FF0000"}
+    FakeWeb.register_uri :put, /http:\/\/localhost:5984\/test\/with_time%2F.+/, body: resp.to_json
+    test = WithTime.new id: "timed"
+    assert test.respond_to?(:created_at)
+    assert test.save
+    assert test.created_at
```

Comment:
You might want to verify `created_at` is the current time rather than just presence.